README.md

```diff
@@ -55,13 +55,14 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 
 ```hcl
 module "kops_external_dns" {
-  source       = "git::https://github.com/cloudposse/terraform-aws-kops-external-dns.git?ref=master"
-  namespace    = "cp"
-  stage        = "prod"
-  name         = "external-dns"
-  cluster_name = "us-east-1.cloudposse.com"
-  masters_name = "masters"
-  nodes_name   = "nodes"  
+  source         = "git::https://github.com/cloudposse/terraform-aws-kops-external-dns.git?ref=master"
+  namespace      = "cp"
+  stage          = "prod"
+  name           = "external-dns"
+  cluster_name   = "us-east-1.cloudposse.com"
```

Comment:
In our examples, we typically use `cloudposse.co` or `example.com`
@aknysh what do you prefer?

Reply:
cloudposse.co